platform/stm32/export/aux/i2c_bus.hpp

```diff
@@ -718,7 +720,38 @@ void i2c_bus<i2c_config>::irq_ev_handler()
 template<class i2c_config>
 void i2c_bus<i2c_config>::irq_er_handler()
 {
-    // TODO add error handling
+    constexpr auto irqn  = pick_er_irqn();
+
+    irq::clear(irqn);
+    constexpr auto i2c = pick_i2c();
+
+    // I2C_GetLastEvent() is suitable when multiple flags are monitored at the same time. Returns type is compatible with I2C_FLAG_* definitions.
```

Comment:
1. line too long, split it according to code style
2. `Returns type is compatible` -> `The return type is`

Reply:
https://forgge.github.io/theCore/community.html#code-style